test/extended/util/disruption/controlplane/controlplane.go

```diff
@@ -118,7 +118,8 @@ func (t *availableTest) Test(f *framework.Framework, done <-chan struct{}, upgra
 
 	toleratedDisruption := 0.08
 	switch {
-	case framework.ProviderIs("azure"), framework.ProviderIs("aws"), framework.ProviderIs("gce"):
+	// framework.ProviderIs("gce") removed here in 4.9 due to regression. BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1981872
+	case framework.ProviderIs("azure"), framework.ProviderIs("aws"):
```

Comment:
Since this previously wasn't tolerated, I don't see why we would add it back.  Let's just do ingress.

Reply:
Fixed